src/pipelines.py

```diff
@@ -0,0 +1,50 @@
+"""
+    Pipelines for cyberowl scraper.
+"""
+from mdtemplate import Template
+
+
+class AlertPipeline:
+    """
+    AlertPipeline class
+    """
+
+    result: list = []
+
+    def remove_special_characters(self, text):
+        """
+        Remove special characters from text.
+        """
+        return (
+            text.replace("\n", "")
+            .replace("\r", "")
+            .replace("\t", "")
+            .replace("  ", "")
+            .replace("|", "")
+        )
+
+    def open_spider(self, spider):
+        """
+        Open spider
```

Comment:
why is this function getting spider as parameter and then setting result to empty list?

Reply:
The function `open_spider` expects it as an argument, however, we can use `*args` and `**kwargs`  instead.